### ISSUE TITLE
3D dynamic analysis is not... dynamic

### ISSUE THREAD

**Issue by cvdr888** [rclab-auth/gidopensees#179]
Hello
I am trying to model 3D bricks of elastic soil subjected to acceleration from a record (Loma Prieta, for example). 
All flowing during the first interval. 
In the sec
[MyModel02.gid.zip](https://github.com/rclab-auth/gidopensees/files/8852051/MyModel02.gid.zip)
ond interval, the analysis is carried out, but it seems that somehow the load was not applied to the elements. 
Can you please verify my model (attachement)? 
Tanh you in advance 
CR.

**Comment by vpapanik** [comment]
You shouldn't apply 'point ground motion from record' when using uniform excitation.
Uniform excitation applies acceleration automatically. 
https://opensees.berkeley.edu/wiki/index.php/Uniform_Excitation_Pattern

**Comment by cvdr888** [comment]
Yes, I understand. However, the model I've sent was my n^th trial. I've tried with and without ground motion point, with and without constraints, with and without soil mass.... etc
I am doing something wrong because the dynamic loading is not being applied. I've also increased the acceleration  to absurd values to verify if something changed in the response displacements....which did not. 
Meanwhile my GidOpenSees license is about to expire. I there any tutorial I can use? I only found the one with the frame-soil dynamic interaction, but none regarding soil-to-structure transmission of displacements. 
Do you have any, please?
Thank you for your help.

**Comment by vpapanik** [comment]
Hello, I am attaching a fixed model. Changes:

[MyModel02_VKP.gid.zip](https://github.com/rclab-auth/gidopensees/files/8992052/MyModel02_VKP.gid.zip)

- changed solver to UmfPack
- fixed boundary conditions at corners (check KNOWN ISSUES)

```
OpenSees does not combine constraints (multiple 'fix' commands on the same node). As a result:

    Intersection joints that belong to two lines with different constraints should be handled separately.
    Intersection lines that belong to two surfaces with different constraints should be handled separately.
```

----> Check again if everything's ok at node level, on meshed model

- fixed boundary conditions on bottom on interval 2 (fix x axis, where record is applied)
- changed time step to 0.01 sec instead of 0.1 sec

most importantly

- added **mass density** to materials, without a mass matrix, the dynamic analysis won' work.
